packages/http-client/src/index.d.ts

```diff
@@ -4,17 +4,20 @@ import type {
   UserProp,
 } from "@webstudio-is/react-sdk";
 
-export type IncludeTypes = "tree" | "props" | "breakpoints" | "pages";
```

Comment:
Just noticed. Why do we have a `.d.ts` file that we edit manually? All this code probably should be in `index.ts`.

But I might be missing something I'm new to TS myself, just seem strange compared to how we do it everywhere else.